ebos/eclproblem.hh

```diff
@@ -2548,7 +2550,7 @@ private:
         eclWriter_->endRestart();
     }
 
-    void processRestartSaturations_(InitialFluidState& elemFluidState)
+    void processRestartSaturations_(InitialFluidState& elemFluidState, size_t elemIdx)
```

Comment:
Before, processRestartSaturations_ could be a const function, it only changed the input elemFluidState. Now it also changes the solventSaturation_ member. I think it would be better to take a `double&` reference here and pass `solventSaturation_[elemIdx]` in the calling code rather than passing `elemIdx` (assuming changing InitialFluidState to contain solvent is a non-starter).

Reply:
Done!